### DISCUSSION TITLE
SEB 3.3.0

### DISCUSSION THREAD

**Discussion by vanie76** [SafeExamBrowser/seb-mac#106] [Q&A]
Can the new version 3.3.0 solve MAC  issues.  The problem we face currently is  integrating ZOOM into SEB for MAC users. The ZOOM application runs as a background application where students have to log into ZOOM before using SEB. The major problem was students who have the latest MAC with Apple M1 chip were unable to install windows. Students used parallel to install windows into MAC with Apple M1 chip. Installing Windows using Parallel, students still have access to MACOS even though they were lock-in SEB in WINDOWS. This means students can still cheat to install windows and use SEB but use MACOS to access other sites. Any solution for this?

**Comment by dbuechel** [comment]
Well, there is a macOS version of SEB, so your students should use that version instead of running SEB for Windows inside a VM on a Macintosh.

**Reply by vanie76** [reply]
can I cofigure the configuration file in the new version 3.3.0 for macOS version by adding  zoom.us in the Application section as shown below
![image](https://user-images.githubusercontent.com/87995554/128017814-840ef0ef-cc0b-496b-9971-297e2ee7025a.png).
Will the macOS student able to see the ZOOM icon . The picture below is SEB in Windows using ZOOM for invigilation
![image](https://user-images.githubusercontent.com/87995554/128018913-a30acf0b-826b-420c-8522-123608542e7b.png)

**Reply by danschlet** [reply]
A later version of SEB for macOS will allow to run additional applications during exams, but only when using macOS 12 Monterey or later (as Apple is adding the feature for additional apps running in the AAC Assessment Mode in Monterey).

But SEB 3.2 for macOS will have Zoom integrated similarly as SEB 3.3 for Windows.

Regarding using Zoom and other WebRTC proctoring services inside the SEB browser see issue #66.

**Reply by vanie76** [reply]
Hi
Manage to configure ZOOM in SEB for macOS with SEB3.3.0
![image](https://user-images.githubusercontent.com/87995554/128054025-ab2e0dd6-2e12-4139-9b59-d6813d7c1d92.png)
Thank you so much for all the help
Truly appreaciated

**Reply by danschlet** [reply]
> Hi
> Manage to configure ZOOM in SEB for macOS with SEB3.3.0

Hmm, did you enable the option 'Allow switching to third party applications'? That option cannot be used if you want to prevent students to use other applications on the Mac, [see the manual](https://safeexambrowser.org/macosx/mac_usermanual_en.html#PermittedProcessesSection):

> Allow switching to third party applications: If this option is activated, the kiosk mode induced by SEB will be reduced one level. The process switcher (Cmd-Tab) will work, so it's possible to switch to other running applications.
Use this option only when running SEB in a managed user account, with only SEB and the desired applications allowed. This will currently only be usable on managed Macs, not in BYOD environments (see Secure Environment).

**Reply by vanie76** [reply]
Yes I enable the feature as mentioned above. What if I block them in Prohibited Processes. We have to integrate the ZOOM app within SEB as we need to communicate with student via chat. Disabling the 'Allowing switching to thrid party application' will only allow ZOOM icon to appear without any functions. If student drop out of the ZOOM due to Internet connectivity they  willl not realise they have dropped out or they  have to exit SEB and rejoin back again. We wanted the MAC users to have the same experience as the Windows users. We give the quit SEB password via chat and all communication with related to invigilation is via chat. Is there any other way I can make this work? I refered to #66 but I didn;t get much help. Sorry :(

![image](https://user-images.githubusercontent.com/87995554/128143207-b34f59a5-5520-4389-a934-4d707e11b0e7.png)
